.github/workflows/coverage.yml

```
@@ -1,33 +1,39 @@
-name: Coverage
+name: CI
 
 on:
   pull_request:
     branches: [main]
 
+variables:
```

actionlint (1.7.7)

unexpected key "variables" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)